samcli/local/docker/manager.py

```diff
@@ -152,6 +157,18 @@ def has_image(self, image_name):
         except docker.errors.ImageNotFound:
             return False
 
+    def _is_rapid_image(self, image_name):
+        """
+        Is the image tagged as a RAPID clone?
+
+        : param string image_name: Name of the image
+        : return bool: True, if the image name ends with rapid-$SAM_CLI_VERSION. False, otherwise
+        """
+
+        if not re.search(r"rapid-\d+\.\d+.\d+$", image_name):
```

Comment:
it is tagged as :rapid-SAMCLIVersion right? should we include the ":"

Reply:
because we are checking the image tag, not the image itself.

Reply:
Changed this to match tag name only.